### PR TITLE
Merge RB-v23.1.0 features into master

### PR DESCRIPTION
**Pull request by BeneBr**

### CODE REVIEW

File: src/alfasim_sdk/_internal/alfacase/case_description_attributes.py [hunk 1]
```diff
@@ -192,6 +192,11 @@ class Foo:
         )
         return Curve(image, domain)
     elif isinstance(value, Curve):
+        if value.GetLength() == 0:
```
Comment (nicoddemus) [inline src/alfasim_sdk/_internal/alfacase/case_description_attributes.py:195]: :eyes:

Comment (Gualcar) [reply src/alfasim_sdk/_internal/alfacase/case_description_attributes.py:195]: Yes, I will create the task to guarantee that it is being tracked.

Only to track the discussion, here is the `alfasim-sdk` PR link https://github.com/ESSS/alfasim-sdk/pull/297.

Comment (Gualcar) [reply src/alfasim_sdk/_internal/alfacase/case_description_attributes.py:195]: Task: ASIM-5237

Comment (nicoddemus) [reply src/alfasim_sdk/_internal/alfacase/case_description_attributes.py:195]: Nice, thanks!

**Pull request merged**